release/RELEASE_PROCESS.md

```diff
@@ -2,48 +2,162 @@
+
+- [ ] [`modelmesh`](https://github.com/kserve/modelmesh/branches)
+- [ ] [`modelmesh-runtime-adapter`](https://github.com/kserve/modelmesh-runtime-adapter/branches)
+- [ ] [`modelmesh-serving`](https://github.com/kserve/modelmesh-serving/branches)
+- [ ] [`rest-proxy`](https://github.com/kserve/rest-proxy/branches)
 
 ## Publish the release
 
-It's generally a good idea to search the repo or control-f for strings of the old version number and replace them with the new, keeping in mind conflicts with other library version numbers.
-
-1. Update `modelmesh`, `modelmesh-runtime-adapter`, `modelmesh-controller`, and `rest-proxy` image tags to the corresponding release version numbers.
-   - Edit `newTag` in `config/manager/kustomization.yaml`.
-   - Edit the `modelmesh`, `modelmesh-runtime-adapter`, and `rest-proxy` image tags in `config/default/config-defaults.yaml`.
-   - Edit the `config/dependencies/quickstart.yaml` file, changing the `kserve/modelmesh-minio-examples` image tag to use the pinned version.
-   - Edit the `docs/component-versions.md` file with the version and component versions.
-   - Edit the `docs/install/install-script.md` file, updating the `RELEASE` variable in the `Installation` section to the new release branch name.
-   - Edit the `docs/quickstart.md` file, updating the `RELEASE` variable in the `Get the latest release` section to the new release branch name.
-   - Edit the `scripts/setup_user_namespaces.sh` file, changing the `modelmesh_release` version.
-1. Submit your PR to the release branch and wait for it to merge.
-1. Update `docs/component-versions.md`, `docs/quickstart.md`, `docs/install/install-script.md`, and `scripts/setup_user_namespaces.sh` files in the main branch with the same versions as above, then submit this as a PR to `main`. Wait for this to merge.
-1. Generate release manifests:
-   - `kustomize build config/default > modelmesh.yaml`
-   - `kustomize build config/runtimes --load-restrictor LoadRestrictionsNone > modelmesh-runtimes.yaml`
-   - `cp config/dependencies/quickstart.yaml modelmesh-quickstart-dependencies.yaml`
-1. Generate config archive:
-   - Run one of the following depending on what version of `tar` you have. Be sure to replace the `update-me` below with the correct version:
-     - GNU tar: `RELEASE=update-me;tar -zcvf config-${RELEASE}.tar.gz config/ --transform s/config/config-${RELEASE}/`
-     - BSD tar: `RELEASE=update-me;tar -zcvf ${RELEASE}.tar.gz -s /config/${RELEASE}/ config/`
-1. Once everything has settled, tag and push the release with `git tag $VERSION` and `git push upstream $VERSION`. You can also tag the release in the GitHub UI.
-   - The `modelmesh-controller` image will be published via GitHub Actions.
-1. Upload generated install manifests to GitHub release assets.
-1. Be sure to create and push a new tag for the `kserve/modelmesh-minio-examples` image.
+It's generally a good idea to search the repo or `control`-`shift`-`f` for strings
```

Comment:
Isn't find ctrl-F (or cmd-F on mac)?

Reply:
I was trying to indicate a search over all files -- control-f (without shift) is a in-file only search